docs/references/pysdk_api_reference.md

```diff
@@ -1783,21 +1788,21 @@ table_object.output(["num", "body", "vec", "sparse", "year", "tensor", "_score"]
             .match_sparse("sparse", {"indices": [0, 20, 80], "values": [1.0, 2.0, 3.0]}, "ip", 3)
             .match("body", "blooms", "topn=10")
             .filter("year < 2024")
-            .fusion("weighted_sum", "topn=2;weights=1,2,0.5")
+            .fusion("weighted_sum", {"topn": 2, "weights": "1,2,0.5"})
             .to_pl()
 ```
 
 #### Use tensor reranking
 
 ```python {8}
-# You must import `CommonMatchTensorExpr`to set tensor reranking parameters
-from infinity.common import CommonMatchTensorExpr
 table_object.output(["num", "body", "vec", "sparse", "year", "tensor", "_score"])
             .knn("vec", [3.0, 2.8, 2.7, 3.1], "float", "cosine", 3)
             .match_sparse("sparse", {"indices": [0, 20, 80], "values": [1.0, 2.0, 3.0]}, "ip", 3)
             .match("body", "blooms", "topn=10")
             .filter("year < 2024")
-            .fusion("match_tensor", "topn=2", commonMatchTensorExpr("tensor", [[0.0, -10.0, 0.0, 0.7], [9.2, 45.6, -55.8, 3.5]], "float", "maxsim"))
+            .fusion("match_tensor", {"topn": 2, "column_name": "tensor",
```

Comment:
1. move topn from dict to fusion parameter.
2. "column_name" => "field"
3. "query_data" => "data"
4. "query_data_type" => "data_type"